server/scripts/setupChainEventListeners.ts

```diff
@@ -21,6 +21,13 @@ const log = factory.getLogger(formatFilename(__filename));
 // emit globally any transfer over 1% of total issuance
 // TODO: config this
 const BALANCE_TRANSFER_THRESHOLD_PERMILL: number = 10_000;
+const BALANCE_TRANSFER_THRESHOLD: number = 10 ** 22;
+/* TODO: both of these are imperfect solutions. The BALANCE_TRANSFER_THRESHOLD_PERMILL used
```

Comment:
The solution I would do would be to fetch and cache the token issuance for each supported token at startup time inside chain events, and use that to compute the percentage whenever we receive an event.

Reply:
Hm good idea, how about we merge this version first and then maybe make new issue to do that as improvement?

Reply:
We could also use the token decimals once the other PR goes in to get an approximation of 1000 tokens.